### ISSUE TITLE
Goals ListView

### ISSUE THREAD

**Issue by awinogradov** [taskany-inc/issues#4]
<img width="630" alt="image" src="https://user-images.githubusercontent.com/982072/163571940-c013af9a-dbd3-4b88-8b41-fe1a41df5349.png">

- [x] #9 
- [x] #19
- [x] #28
- [x] pagination

**Comment by awinogradov** [comment]
### Tag color randomizer

``` js
const ColorContrastChecker = require("color-contrast-checker");
const ccc = new ColorContrastChecker();

// default background
const backgroundColor = `#1b1b1c`;
// contrast ratio
const ratio = 6;

const genColor = () => `#${Math.floor(Math.random()*16777215).toString(16)}`;
let generatedColor;
let valid = false;

while (!valid) {
    generatedColor = genColor();
    try {
        valid = ccc.isLevelCustom(generatedColor, backgroundColor, ratio);
    } catch (e) {
        valid = ccc.isLevelCustom(generatedColor, backgroundColor, ratio);
    }
}

console.log(backgroundColor, generatedColor, valid)
```

**Comment by awinogradov** [comment]
Resolves in #75
